### PR TITLE
Edit text in autotune config pane.

### PR DESCRIPTION
**Pull request by ufoDziner**
Added a bit more description no bullet number 8.

[Review on Reviewable](https://reviewable.io/reviews/d-ronin/dronin/293)

### CODE REVIEW

**Comment by mlyle** [comment]
jenkins, test this please

**Comment by mlyle** [comment]
Happy to merge with a screenshot, or will get to this myself sometime relatively soon.

**Comment by mlyle** [comment]
jenkins, test this please

File: ground/gcs/src/plugins/config/autotune.ui [hunk 1]
```diff
@@ -82,23 +82,23 @@ p, li { white-space: pre-wrap; }
...
+&lt;li style=&quot; font-family:'Lucida Grande'; font-size:13pt;&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Adjust the values as necessary with the "Damping" and "Noise sensitivity" sliders until they appear to be reasonable. When satisfied, click the "Apply Computed Values" button at the bottom right.&lt;br /&gt;&lt;/li&gt;
+&lt;li style=&quot; font-family:'Lucida Grande'; font-size:13pt;&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Retun to the &lt;span style=&quot; font-style:italic;&quot;&gt;Stabilization&lt;/span&gt; configuration screen and click "apply" to enable the new settings. The "save" button must also be pressed if the flight controller is to be powered down prior to the next flight.&lt;br /&gt;&lt;/li&gt;
```
Comment (mlyle) [inline ground/gcs/src/plugins/config/autotune.ui]: "Return"

File: ground/gcs/src/plugins/config/autotune.ui [hunk 2]
```diff
@@ -82,23 +82,23 @@ p, li { white-space: pre-wrap; }
...
-&lt;p style=&quot; margin-top:0px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Lucida Grande'; font-size:13pt;&quot;&gt;Once you have understood the above steps, begin the procedure. Advanced users can proceed to the &lt;/span&gt;&lt;span style=&quot; font-family:'Lucida Grande'; font-size:13pt; font-style:italic;&quot;&gt;Autotune Setup&lt;/span&gt;&lt;span style=&quot; font-family:'Lucida Grande'; font-size:13pt;&quot;&gt; tab for more options.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+&lt;li style=&quot; font-family:'Lucida Grande'; font-size:13pt;&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Land and disarm leaving the Autotune mode enabled. At this point, the settings are saved. Power down the UAV, unless it is connected to GCS via bluetooth&lt;br /&gt;&lt;/li&gt;
```
Comment (mlyle) [inline ground/gcs/src/plugins/config/autotune.ui]: period at end of sentence

File: ground/gcs/src/plugins/config/autotune.ui [hunk 3]
```diff
@@ -82,23 +82,23 @@ p, li { white-space: pre-wrap; }
...
+&lt;li style=&quot; font-family:'Lucida Grande'; font-size:13pt;&quot; style=&quot; margin-top:0px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Ensure that a normal flight mode (not autotune) is selected prior to flight.  If autotune is left as the active mode, it is likely that the flight controller will not arm.&lt;/li&gt;
+&lt;li style=&quot; font-family:'Lucida Grande'; font-size:13pt;&quot; style=&quot; margin-top:0px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;If the UAV performs well, then the process is complete.  If it does not, the sliders may be ajusted, settings reapplied, saved and tested again.&lt;/li&gt;&lt;/ol&gt;
```
Comment (mlyle) [inline ground/gcs/src/plugins/config/autotune.ui]: Missing closing </string> tag

Comment (mlyle) [inline ground/gcs/src/plugins/config/autotune.ui]: "adjusted"

**Comment by mlyle** [comment]
jenkins, test this please

**Comment by mlyle** [comment]
Tested; this looks pretty good.  Will add a couple comments.

File: ground/gcs/src/plugins/config/autotune.ui [hunk 4]
```diff
@@ -82,23 +82,23 @@ p, li { white-space: pre-wrap; }
...
+&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Lucida Grande'; font-size:13pt;&quot;&gt;This is an advanced plugin for the GCS that is going to make your aircraft twitch automatically, so test in a large open area and be &lt;/span&gt;&lt;span style=&quot; font-family:'Lucida Grande'; font-size:13pt; font-weight:600;&quot;&gt;extremely wary&lt;/span&gt;&lt;span style=&quot; font-family:'Lucida Grande'; font-size:13pt;&quot;&gt; for it creating bad tuning values.&lt;br /&gt;&lt;br /&gt;Please read and understand all of the following steps for autotuning before attempting an autotune.&lt;br /&gt;&lt;/span&gt;&lt;/p&gt;
```
Comment (mlyle) [inline ground/gcs/src/plugins/config/autotune.ui]: extremely wary for ~~it creating~~ bad tuning values.

File: ground/gcs/src/plugins/config/autotune.ui [hunk 5]
```diff
@@ -82,23 +82,23 @@ p, li { white-space: pre-wrap; }
...
+&lt;li style=&quot; font-family:'Lucida Grande'; font-size:13pt;&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Land and disarm leaving the Autotune mode enabled. At this point, the settings are saved. Power down the UAV, unless it is connected to GCS via bluetooth.&lt;br /&gt;&lt;/li&gt;
+&lt;li style=&quot; font-family:'Lucida Grande'; font-size:13pt;&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Connect to GCS and check the values on the &lt;span style=&quot; font-style:italic;&quot;&gt;Autotune Setup&lt;/span&gt; page against the values on the &lt;span style=&quot; font-style:italic;&quot;&gt;Stabilization&lt;/span&gt; configuration screen. If they are vastly different, this is an indication to be wary, and indicates that a subsequent autotune should be run to get more accurate values.&lt;span style=&quot; font-weight:600;&quot;&gt;For extreme values a prompt will ask if you want to continue. Check your settings carefully in this case.&lt;/span&gt;&lt;br /&gt;&lt;/li&gt;
```
Comment (mlyle) [inline ground/gcs/src/plugins/config/autotune.ui]: Add space before "For extreme values"

File: ground/gcs/src/plugins/config/autotune.ui [hunk 6]
```diff
@@ -82,23 +82,23 @@ p, li { white-space: pre-wrap; }
...
+&lt;li style=&quot; font-family:'Lucida Grande'; font-size:13pt;&quot; style=&quot; margin-top:0px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Ensure that a normal flight mode (not autotune) is selected prior to flight.  If autotune is left as the active mode, it is likely that the flight controller will not arm.&lt;/li&gt;
+&lt;li style=&quot; font-family:'Lucida Grande'; font-size:13pt;&quot; style=&quot; margin-top:0px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;If the UAV performs well, then the process is complete.  If it does not, the sliders may be adjusted, settings reapplied, saved and tested again.&lt;/li&gt;&lt;/ol&gt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```
Comment (mlyle) [inline ground/gcs/src/plugins/config/autotune.ui]: Missing &lt; before last /span

Comment (mlyle) [inline ground/gcs/src/plugins/config/autotune.ui]: &amp;lt;  that is (github/markdown changed it to an <)

**Comment by mlyle** [comment]
Thanks for doing this.  I know manually editing .ui files is no fun (in github, no less)

**Comment by ufoDziner** [comment]
Just added an updated version.  I'll get my git skills up to speed soon.

**Comment by mlyle** [comment]
Congrats on your first PR :)

**Pull request merged**